snapshots/emissions/2023-10-24/emission_factors.py

```diff
@@ -0,0 +1,32 @@
+"""Script to create a snapshot of dataset.
+
+To create a snapshot of the dataset, follow these steps:
+* Go to https://www.ipcc-nggip.iges.or.jp/EFDB/
+* Click on the button "Export to XLS".
```

Comment:
I cannot see this button

Reply:
Is this the link instead? https://www.ipcc-nggip.iges.or.jp/EFDB/find_ef.php?reset=